src/supertux/sector.hpp

```diff
@@ -49,6 +50,14 @@ class TextObject;
 class TileMap;
 class Writer;
 
+namespace {
```

Comment:
What is this empty namespace doing here

Reply:
In case you don't have enough names already and need space for more. 

Like for example: Albus Percival Wulfric Brian [... enter more names here] Dumbledore.